src/ray/protobuf/monitor.proto

```diff
@@ -40,4 +104,6 @@ service MonitorGcsService {
   // Request that GCS drain and kill a node. This call is idempotent, and could
   // need to be retried if the head node fails.
   rpc DrainAndKillNode(DrainAndKillNodeRequest) returns (DrainAndKillNodeReply);
+
+  rpc GetSchedulingStatus(GetSchedulingStatusRequest) returns (GetSchedulingStatusReply);
```

Comment:
Have we ever considered the size of the messages? Could it be too big in a single request?

Reply:
I believe this is strictly smaller than the current message we're using and empirically it hasn't been a problem.

Reply:
I think I got some issues when I run the test with > 3k nodes and it's said message is too big. :(